feature-utils/poly-look/src/react-components/infographic/infographic.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import "./infographic.css";
+
+/** Renders an image and a numbered legend that contains descriptions
+ *  for different elements in the image.
+ * @param {Object} props
+ * @param {string} [props.imageSrc] - src of the image we want to render
+ * @param {Array[string]} [props.legend] - a simple array with descriptions
+ * @returns {JSX.Element}
+ */
+const Infographic = ({ imageSrc, legend }) => {
+  return (
+    <div className="poly-infographic">
+      <div className="image-container">
+        <img src={imageSrc} />
+      </div>
+      <div className="legend-container">
+        {legend.map((entry, idx) => {
+          return (
+            <div
+              className="entry"
+              key={idx}
+              data-testid="infographic-entry-test"
+            >
+              <div className="number">
+                <span>{idx + 1}</span>
```

Comment:
It's a bit confusing that the index and the actual number shown are different. Maybe make them the same?

Reply:
Initially they were the same, but then I've realized that in the design the numbering always starts at 1.